pkg/api/apis/operators/v1/operatorgroup_types.go

```diff
@@ -28,8 +28,9 @@ type OperatorGroupSpec struct {
 	// +optional
 	TargetNamespaces []string `json:"targetNamespaces,omitempty"`
 
-	// ServiceAccount to bind OperatorGroup roles to.
-	ServiceAccount corev1.ServiceAccount `json:"serviceAccount,omitempty"`
+	// ServiceAccountName is the admin specified service account which will be
+	// used to deploy operator(s) in this operator group.
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```

Comment:
The corresponding  schema in `deploy/chart` and `manifests` should be updated to match this

Reply:
This is already set to `serviceAccountName` - https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/chart/templates/0000_50_olm_10-operatorgroup.crd.yaml#L78.